webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js

```diff
@@ -167,7 +167,7 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
         </div>
       </div>
       <div class="grader-navigator">
-        <div><a class="user-list-link" href="${this.userListUrl}" title="${this.assignmentsI18n["nav.list"]}">${this.assignmentsI18n["nav.list"]}</a></div>
+        <div><a class="user-list-link" href="javascript:void(0)" title="${this.assignmentsI18n["nav.list"]}" @click=${this.toStudentList}>${this.assignmentsI18n["nav.list"]}</a></div>
```

Comment:
i always prefer a -button- when i see javascript:void

Reply:
Currently this is showing a regular link (underlining blue text), our btn-link css classes shows it as normal text. Probably change this link and putting a normal button could be a good idea, because old grader shows a normal button.

Reply:
Finally I changed the link by a button, with the css now it looks like a button. How do yo see it @adrianfish @ottenhoff? 
![link-to-button](https://user-images.githubusercontent.com/19774177/156521343-26d3a32d-3d41-4fe4-b202-6e1b70da6a01.png)

Reply:
> i always prefer a -button- when i see javascript:void

@ottenhoff  But this link causes a navigation, not a popup. Aren't links fine in this situation?

Reply:
Ok here is a good rule of thumb:

"Buttons are used for actions that affect the website’s front-end or back-end; links are used for navigation and actions that don’t affect the website at all."

https://ux.iu.edu/writings/buttons-vs-links-basic/#:~:text=There%20are%20differences%20as%20to,affect%20the%20website%20at%20all.

I guess I get suspicious when I see javascript in a link as it usually indicates that *navigation* is *not* being performed.